VogtPayroll3/Employee.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace VogtPayroll3
+{
+    class Employee
+    {
+        // Fields
+        private string _firstName;
+        private string _lastName;
+        private int _empID;
+        private int _hoursWorked;
+        private decimal _payrate;
+        private const decimal taxAmount = .30m;
+
+        public string FirstName { get => _firstName; set => _firstName = value; }
+        public string LastName { get => _lastName; set => _lastName = value; }
+        public int EmpID { get => _empID; set => _empID = value; }
+        public int HoursWorked { get => _hoursWorked; set => _hoursWorked = value; }
+        public decimal Payrate { get => _payrate; set => _payrate = value; }
+
+        // CTOR
+        public Employee(string firstName, string lastName, int empID, int hoursWorked, decimal payRate)
+        {
+            this.FirstName = firstName;
+            this.LastName = lastName;
+            this.EmpID = empID;
+            this.HoursWorked = hoursWorked;
+            this.Payrate = payRate;
+
+        }
+
+        public Employee CreateAnEmployee()
```

Comment:
You need to get away from having objects instantiate themselves like this. It's a class method which creates another instance of itself. Remember before how you had a PayrollConsoleReader which would build the employee for you? Try making this more like the last one you built where the console reader abstracts building a new employee.

**The employee object should never be responsible for creating a new employee, UNLESS its a `Copy()` method, which this is not.**